cumulus/steps/ec2/launch_config.py

```diff
@@ -0,0 +1,81 @@
+from troposphere import autoscaling, Ref, FindInMap, Base64, ec2
+from cumulus.chain import step
+import cumulus.components.userdata.linux
+from cumulus.steps.ec2 import META_SECURITY_GROUP_NAME
+
+
+class LaunchConfig(step.Step):
+
+    def __init__(self,
+                 asg_name,
+                 launch_config_name,
+                 meta_data,
+                 instance_profile_name,
+                 vpc_id=None,
+                 user_data=None, ):
+
+        step.Step.__init__(self)
+
+        self.asg_name = asg_name
+        self.launch_config_name = launch_config_name
+        self.user_data = user_data
+        self.meta_data = meta_data
+        self.instance_profile_name = instance_profile_name
+        self.vpc_id = vpc_id
+
```

Comment:
It seems like  the following could be derived, and at least made optional here, to simplify the consumer requirements. 
* asg_name
* launch_config_name
* instance_profile_name